### PR TITLE
fix(discord): Process `delete channel & restart import` button

### PR DESCRIPTION
**Pull request by borismelnik**
Fixes: #12684

### What does the PR do

Process `delete channel & restart import` button

### Affected areas

Discord import single channel

### StatusQ checklist

- [ ] add documentation if necessary (new component, new feature)
- [ ] update sandbox app
  - in case of new component, add new component page
  - in case of new features, add variation to existing component page
  - nice to have: add it to the demo application as well
- [ ] test changes in both light and dark theme?

### CODE REVIEW

**Comment by borismelnik** [comment]
this PR waiting for [STATUSGO](https://github.com/status-im/status-go/pull/4278), because there I fix Go auto replace `string(request.CommunityID)` to `request.CommunityID.String()` (unfortunately, its not the  same)

**Review by jrainville — APPROVED** [review]
LGTM. You say it depends on status-go, but the vendor isn't updated :thinking:

**Comment by borismelnik** [comment]
> LGTM. You say it depends on status-go, but the vendor isn't updated 🤔

yeah I don't want to point the same status-go commit to two different status-desktop PRs. :) I should merge this one first https://github.com/status-im/status-desktop/pull/12655 and when it will be merged this functionality should works correct :)

File: src/app/modules/main/communities/view.nim [hunk 1]
```diff
@@ -634,6 +652,11 @@ QtObject:
     self.delegate.requestCancelDiscordChannelImport(discordChannelId)
     self.resetDiscordImport(true)
 
+  proc removeImportedDiscordChannel*(self: View) {.slot.} =
+    echo "LALALA ", self.discordImportedChannelCommunityId, " " , self.discordImportedChannelId
```
Comment (caybro) [inline src/app/modules/main/communities/view.nim:656]: debug leftover

File: src/app/modules/main/communities/view.nim [hunk 2]
```diff
@@ -50,6 +50,8 @@ QtObject:
       discordImportChannelId: string
       discordImportChannelName: string
       discordImportCommunityImage: string
+      discordImportedChannelId: string
```
Comment (caybro) [inline src/app/modules/main/communities/view.nim:53]: Do we really need a separate property for this? We now have `discordImportChannelId` and `discordImportedChannelId`, easy to confuse imnsho

File: ui/app/AppLayouts/Communities/popups/DiscordImportProgressContents.qml [hunk 3]
```diff
@@ -364,4 +368,28 @@ StatusScrollView {
             }
         }
     }
+
+    Component {
+        id: deleteAndRestartConfirmationPopupCmp
+        ConfirmationDialog {
+            id: deleteAndRestartConfirmationPopup
+            headerSettings.title: qsTr("Are you sure you want to delete the channel?")
+            confirmationText: qsTr("Your new Status %1 will be deleted and all information entered will be lost.").arg(root.importingSingleChannel ? qsTr("channel") : qsTr("community"))
```
Comment (caybro) [inline ui/app/AppLayouts/Communities/popups/DiscordImportProgressContents.qml:377]: This should be just "channel", there's a TODO for "community" above and "Delete channel" below

File: src/app/modules/main/communities/view.nim (hunk 2 above)
Comment (borismelnik) [reply src/app/modules/main/communities/view.nim:53]: I created the issue to refactoring this :)

**Pull request merged**